src/charm.py

```diff
@@ -28,7 +28,7 @@
 # Juju defines the value of this label.
 # It has the same value as the label "app.kubernetes.io/name"
 # set in the service account associated with the application.
-CREATED_BY_LABEL = "app.juju.is/created-by="
+CREATED_BY_LABEL = "Ingress-label"
```

Comment:
Should the variable be renamed now that the label is no longer `created-by`? Also, minor thing, should it be `ingress-label`, so be all lower case?